lib/Scvmm/miq_hyperv_disk.rb

```diff
@@ -0,0 +1,46 @@
+
```

Comment:
which of the requires below is coming from util? if none, then we do not need this push.